jbmc/src/janalyzer/janalyzer_parse_options.h

```diff
@@ -174,7 +173,7 @@ class janalyzer_parse_optionst : public parse_options_baset
 
   ui_message_handlert::uit get_ui()
   {
-    return ui_message_handler.get_ui();
+    return message_handler.get_ui();
```

Comment:
Is this method (`janalyzer_parse_optionst::get_ui`) actually used?

Reply:
I think it's not useful anyway...

Reply:
Maybe just add a commit that removes those?